frontend/src/GuessControllerTestConfig.ts

```diff
@@ -0,0 +1,13 @@
+import { ViewGameInProgress } from "./controllers/models/ViewGameInProgress";
+import { BoundaryGameInProgress } from "./use-cases/model/BoundaryGameInProgress";
+
+export const expectedUCValue = new BoundaryGameInProgress(
```

Comment:
Sakyčiau kad neverta taip iškėlinėti. Labai apsišiukšlins failais nereikalingais, o testai nuo to bent man asmeniškai tik pasunkėja, nes duomenys neaišku kur ir kokie sukuriami